<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>1097</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Message</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>60</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>451</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Say it!</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_Run_Main">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Main</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Run Main NZB Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_write_to_textedit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_textbox_string">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_textbox_string">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_get_value_from_table">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get value from table</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>adadsfdafsdfsa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>afdfas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>435</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2345</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2345</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listWidget_movie_type">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>test item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Another test item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>asdfasdfasdf</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>asdfasdf</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>asdf</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_reload_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload Table</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_file_password">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>551</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Filename</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_load_password">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_get_value_list_widget">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Status&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_bus_exp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>1141</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Business Expenses</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_sid"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Date (mm/dd/yyyy):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_sid">
       <property name="text">
        <string>Series ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_date"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_save_params">
       <property name="text">
        <string>Save Params</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_status_clear">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_nzb_series">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>1141</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>New Column 0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column 11</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_nzb_series">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Series</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nzb_series_clear">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>540</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Table</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
